go/oasis-test-runner/scenario/e2e/txsource.go

```diff
@@ -0,0 +1,53 @@
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/env"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/oasis"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/scenario"
+)
+
+var TxSource scenario.Scenario = &txSourceImpl{basicImpl{
+	name:         "txsource",
+	clientBinary: "txsource-wrapper.sh",
+}}
+
+type txSourceImpl struct {
+	basicImpl
+}
+
+func (sc *txSourceImpl) Fixture() (*oasis.NetworkFixture, error) {
+	f, err := sc.basicImpl.Fixture()
+	if err != nil {
+		return nil, err
+	}
+	f.Network.StakingGenesis = "tests/fixture-data/txsource/staking-genesis.json"
```

Comment:
Maybe explicitly say (in a comment) that this fixture will only work with the default seed?

Reply:
I'll add a readme to the fixture directory

Reply:
:+1: